src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function getSum(firstValue) {
+    return function(secondValue) {
+      counter++;
+
+      return (counter > 3 && counter % 2 === 0)
+        ? `Bzzz... Error!`
+        : Number(firstValue) + Number(secondValue);
```

Comment:
Dont really sure that you need to use `Number()`

Reply:
But according to the condition, it is said that we receive numbers) You edited the tests yourself, but ok)